angular-2-components/src/app/shared/components/info/info.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="container">
+  <ng-template [ngIf]="isShowMessage">
+    <h2 class="title">{{ option.title }}</h2>
+    <p class="message" [innerHTML]="option.message"></p
```

Comment:
`[innerHTML]="option.message"` why do you use this one instead simple `<p> {{option.message}} </p>`?